### PR TITLE
Update summary of the tasks

### PR DESCRIPTION
**Pull request by stevhliu**
This PR updates the Summary of the tasks page with audio classification, ASR, and image classification.

### CODE REVIEW

File: docs/source/task_summary.mdx [hunk 1]
```diff
@@ -967,3 +967,158 @@ Here is an example of doing translation using a model and a tokenizer. The proce
 </frameworkcontent>
 
 We get the same translation as with the pipeline example.
+
+## Audio classification
+
+Audio classification assigns a class to an audio signal. The Keyword Spotting dataset from the [SUPERB](https://huggingface.co/datasets/superb) benchmark is an example dataset that can be used for audio classification fine-tuning. This dataset contains ten classes of keywords for classification. If you'd like to fine-tune a model for audio classification, take a look at the [run_audio_classification.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/audio-classification/run_audio_classification.py) script or the how-to guide [here](./tasks/audio_classification).
```
Comment (sgugger) [inline docs/source/task_summary.mdx:973]:
```suggestion
Audio classification assigns a class to an audio signal. The Keyword Spotting dataset from the [SUPERB](https://huggingface.co/datasets/superb) benchmark is an example dataset that can be used for audio classification fine-tuning. This dataset contains ten classes of keywords for classification. If you'd like to fine-tune a model for audio classification, take a look at the [run_audio_classification.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/audio-classification/run_audio_classification.py) script or this [how-to guide](./tasks/audio_classification).
```

File: docs/source/task_summary.mdx [hunk 2]
```diff
@@ -967,3 +967,158 @@ Here is an example of doing translation using a model and a tokenizer. The proce
...
+[{'label': 'calm', 'score': 0.13856211304664612},
+ {'label': 'disgust', 'score': 0.13148026168346405},
+ {'label': 'happy', 'score': 0.12635163962841034},
+ {'label': 'angry', 'score': 0.12439591437578201},
+ {'label': 'fearful', 'score': 0.12404385954141617}]
+```
+
+The general process for using a model and tokenizer for audio classification is:
+
+1. Instantiate a tokenizer and a model from the checkpoint name.
```
Comment (sgugger) [inline docs/source/task_summary.mdx:993]:
```suggestion
1. Instantiate a feature extractor and a model from the checkpoint name.
```
The tokenizer is to decode model predictions only, so not necessary in audio classification.

File: docs/source/task_summary.mdx [hunk 3]
```diff
@@ -967,3 +967,158 @@ Here is an example of doing translation using a model and a tokenizer. The proce
...
+>>> predicted_class_ids = torch.argmax(logits, dim=-1).item()
+>>> predicted_label = model.config.id2label[predicted_class_ids]
+>>> predicted_label
+```
+</pt>
+</frameworkcontent>
+
+## Automatic speech recognition
+
+Automatic speech recognition transcribes an audio signal to text. The [Common Voice](https://huggingface.co/datasets/common_voice) dataset is an example dataset that can be used for automatic speech recognition fine-tuning. It contains an audio file of a speaker and the corresponding sentence. If you'd like to fine-tune a model for automatic speech recognition, take a look at the [run_speech_recognition_ctc.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_ctc.py) or [run_speech_recognition_seq2seq.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py) scripts or the how-to guide [here](./tasks/asr).
```
Comment (sgugger) [inline docs/source/task_summary.mdx:1026]:
```suggestion
Automatic speech recognition transcribes an audio signal to text. The [Common Voice](https://huggingface.co/datasets/common_voice) dataset is an example dataset that can be used for automatic speech recognition fine-tuning. It contains an audio file of a speaker and the corresponding sentence. If you'd like to fine-tune a model for automatic speech recognition, take a look at the [run_speech_recognition_ctc.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_ctc.py) or [run_speech_recognition_seq2seq.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py) scripts or this [how-to guide](./tasks/asr).
```

File: docs/source/task_summary.mdx [hunk 4]
```diff
@@ -967,3 +967,158 @@ Here is an example of doing translation using a model and a tokenizer. The proce
...
+>>> speech_recognizer = pipeline(
+...     task="automatic-speech-recognition", model="facebook/wav2vec2-base-960h"
+... )
+>>> speech_recognizer("jfk_moon_speech.wav")
+{'text': "PRESENTETE MISTER VICE PRESIDENT GOVERNOR CONGRESSMEN THOMAS SAN O TE WILAN CONGRESSMAN MILLA MISTER WEBB MSTBELL SCIENIS DISTINGUISHED GUESS AT LADIES AND GENTLEMAN I APPRECIATE TO YOUR PRESIDENT HAVING MADE ME AN HONORARY VISITING PROFESSOR AND I WILL ASSURE YOU THAT MY FIRST LECTURE WILL BE A VERY BRIEF I AM DELIGHTED TO BE HERE AND I'M PARTICULARLY DELIGHTED TO BE HERE ON THIS OCCASION WE MEED AT A COLLEGE NOTED FOR KNOWLEGE IN A CITY NOTED FOR PROGRESS IN A STATE NOTED FOR STRAINTH AN WE STAND IN NEED OF ALL THREE"}
+```
+
+The general process for using a model and tokenizer for automatic speech recognition is:
+
+1. Instantiate a tokenizer and a model from the checkpoint name.
```
Comment (sgugger) [inline docs/source/task_summary.mdx:1042]:
```suggestion
1. Instantiate a processor (which regroups a feature extractor for input processing and a tokenizer for decoding) and a model from the checkpoint name.
```

File: docs/source/task_summary.mdx [hunk 5]
```diff
@@ -967,3 +967,158 @@ Here is an example of doing translation using a model and a tokenizer. The proce
...
+
+>>> transcription = processor.batch_decode(predicted_ids)
+>>> transcription[0]
+```
+</pt>
+</frameworkcontent>
+
+## Image classification
+
+Like text and audio classification, image classification assigns a class to an image. The [CIFAR-100](https://huggingface.co/datasets/cifar100) dataset is an example dataset that can be used for image classification fine-tuning. It contains an image and the corresponding class. If you'd like to fine-tune a model for image classification, take a look at the [run_image_classification.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/image-classification/run_image_classification.py) script or the how-to guide [here](./tasks/image_classification).
```
Comment (sgugger) [inline docs/source/task_summary.mdx:1074]:
```suggestion
Like text and audio classification, image classification assigns a class to an image. The [CIFAR-100](https://huggingface.co/datasets/cifar100) dataset is an example dataset that can be used for image classification fine-tuning. It contains an image and the corresponding class. If you'd like to fine-tune a model for image classification, take a look at the [run_image_classification.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/image-classification/run_image_classification.py) script or [this how-to guide](./tasks/image_classification).
```

File: docs/source/task_summary.mdx [hunk 6]
```diff
@@ -967,3 +967,158 @@ Here is an example of doing translation using a model and a tokenizer. The proce
...
+  'score': 0.03433405980467796},
+ {'label': 'snow leopard, ounce, Panthera uncia',
+  'score': 0.032148055732250214},
+ {'label': 'Egyptian cat', 'score': 0.02353910356760025},
+ {'label': 'tiger cat', 'score': 0.023034192621707916}]
+```
+
+The general process for using a model and tokenizer for image classification is:
+
+1. Instantiate a tokenizer and a model from the checkpoint name.
```
Comment (sgugger) [inline docs/source/task_summary.mdx:1096]:
```suggestion
The general process for using a model and feature extractor for image classification is:

1. Instantiate a feature extractor and a model from the checkpoint name.
```

**Review by sgugger — APPROVED** [review]
Thanks for adding those examples!
